uber/models.py

```diff
@@ -648,6 +648,9 @@ def change_badge(self, attendee, badge_type, badge_num=None):
         def valid_attendees(self):
             return self.query(Attendee).filter(Attendee.badge_status != c.INVALID_STATUS)
 
+        def all_attendees(self):
+            return self.staffers(only_staffing=False)
+
```

Comment:
I was really confused at a quick glance looking at the email sending code how ```session.staffers(only_staffing=False)``` was getting all attendees until I read the comment.  To just make this a bit clearer, I made this shortcut for ```session.all_attendees()``` here which does the same thing but is more obvious what it's doing.